frontend-react/src/components/CodeMapping/CodeMappingResults.tsx

```diff
@@ -1,10 +1,61 @@
-import type { PropsWithChildren } from "react";
+import { Button } from "@trussworks/react-uswds";
+import { CodeMapData } from "../../hooks/api/UseCodeMappingFormSubmit/UseCodeMappingFormSubmit";
+import { Alert, Table } from "../../shared";
 
-export type CodeMappingResultsProps = PropsWithChildren;
+interface CodeMappingResultsProps {
+    fileName: string;
+    data: CodeMapData[];
+    initialStepHandler: () => void;
```

Comment:
nit: maybe rename to onReset, if that's what is practically happening on button usage